src/vm/exceptionhandling.cpp

```diff
@@ -4424,7 +4428,11 @@ VOID UnwindManagedExceptionPass2(EXCEPTION_RECORD* exceptionRecord, CONTEXT* unw
                 dispatcherContext.FunctionEntry,
                 callerFrameContext,
                 &handlerData,
+#ifdef _ARM_
+                (PULONG)&establisherFrame,
```

Comment:
Is this a PULONG on AMD64 as well?  If not can we unify this?  Else, why is it upcasting?